packages/nimble-components/src/table/tests/table.stories.ts

```diff
@@ -13,13 +13,23 @@ interface TableArgs {
 const simpleData = [
     {
         myStr: 'my first row',
-        myNum: 5,
-        myBool: true
+        myNum: '5',
+        myBool: 'true'
     },
     {
         myStr: 'my second row',
-        myNum: 15,
-        myBool: false
+        myNum: '15',
+        myBool: 'false'
+    },
+    {
+        myStr: 'This is an absurd amount of text to illustrate column behavior with so much text',
```

Comment:
Should we be ellipsizing or truncating long fields? In Firefox I'm seeing it truncated but expected it to be ellipsized (just a personal opinion, not my call, just want to make sure a designer thought about it and we're following their guidance).

Reply:
We should be ellipsizing...this got lost when I updated the template to use a container div to host the cell content.